RadixWallet/Features/AssetsFeature/Components/PoolUnitsList/Components/PoolUnit+View.swift

```diff
@@ -53,41 +35,39 @@ extension PoolUnit {
 extension PoolUnit.State {
 	var viewState: PoolUnit.ViewState {
 		.init(
-			iconURL: poolUnit.resource.metadata.iconURL,
-			name: poolUnit.resource.metadata.name ?? L10n.Account.PoolUnits.unknownPoolUnitName,
+			poolName: poolUnit.resource.metadata.fungibleResourceName,
+			dAppName: resourceDetails.dAppName,
+			poolIcon: poolUnit.resource.metadata.iconURL,
 			resources: resourceDetails.map { details in
-				PoolUnitResourceViewState.viewStates(amount: poolUnit.resource.amount, resourcesDetails: details)
+				PoolUnitResourceView.ViewState.viewStates(resourcesDetails: details)
 			},
 			isSelected: isSelected
 		)
 	}
 }
 
-extension PoolUnitResourceViewState {
+extension PoolUnitResourceView.ViewState {
```

Comment:
This could perhaps benefit from being an extension on `[PoolUnitResourceView.ViewState]`